client/views/room/Header/RoomHeader.tsx

```diff
@@ -13,7 +14,7 @@ import Favorite from './icons/Favorite';
 import Translate from './icons/Translate';
 
 export type RoomHeaderProps = {
-	room: IOmnichannelRoom;
+	room: IOmnichannelRoom & IOmnichannelSubscription;
```

Comment:
Why do we need to support IOmnichannelSubscription interface here given that useOmnichannelRoom always return just IRoom ?